tastyworks/example.py

```diff
@@ -1,73 +1,212 @@
 import asyncio
 import calendar
 import logging
+import pandas as pd
 from os import environ
-from datetime import date, timedelta
+from datetime import date, timedelta, time, datetime
 from decimal import Decimal
 
 from tastyworks.models import option_chain, underlying
 from tastyworks.models.option import Option, OptionType
-from tastyworks.models.order import (Order, OrderDetails, OrderPriceEffect,
-                                     OrderType)
+from tastyworks.models.greeks import Greeks
+from tastyworks.models.quote import Quote
+from tastyworks.models.order import (Order, OrderDetails, OrderPriceEffect, OrderType)
 from tastyworks.models.session import TastyAPISession
 from tastyworks.models.trading_account import TradingAccount
 from tastyworks.models.underlying import UnderlyingType
 from tastyworks.streamer import DataStreamer
 from tastyworks.tastyworks_api import tasty_session
 
+from tastyworks.dxfeed.mapped_item import MappedItem
+
 LOGGER = logging.getLogger(__name__)
 
 
 async def main_loop(session: TastyAPISession, streamer: DataStreamer):
-    # sub_values = {
-    #     "Greeks": [
-    #         ".VIX180718C21",
-    #         ".YUM180518C95"
-    #     ]
-    # }
-    sub_values = {
-        "Quote": ["/ES"]
-    }
 
+    ###############
+    # OPEN ORDERS #
+    ###############
+    # Get accounts
+    print("--------------")
     accounts = await TradingAccount.get_remote_accounts(session)
-    acct = accounts[0]
     LOGGER.info('Accounts available: %s', accounts)
+    print("--------------")
 
+    # Get & list all open orders
+    acct = accounts[0]
     orders = await Order.get_remote_orders(session, acct)
+    print("--------------")
     LOGGER.info('Number of active orders: %s', len(orders))
+    for order in orders:
+        print(order.details)
+    print("--------------")
 
-    # Execute an order
+    ############
+    # EQUITIES #
+    ############
+    # Get quotes of equities
+    symbols = ["SPY", "AAPL", "SPX"]
+    streamer_list = {"Quote": symbols}
+    await streamer.add_data_sub(streamer_list)
+    # fetch the current data one time via streamer
+    streamer_data = []
+    async for item in streamer.listen():
+        # LOGGER.info('Received item: %s' % item.data)
+        streamer_data.extend(item.data)
+        if streamer_data.__len__() == streamer_list['Quote'].__len__():
+            await streamer.reset_data_subs()
+            break  # Stops the async for item in streamer.listen() loop after receiving all the data
+
+    # Store in a list
+    print("--------------")
+    quotes = []
+    for data in streamer_data:
+        qd = Quote(data)
+        quotes.append(qd)
+        print(f"Ticker: {qd.symbol}, Bid: {qd.bid_price}, Ask: {qd.ask_price}")
+    print("--------------")
+
+    # Storing in a dictionary nested with datetime string instead
+    # Not a great way to store data, see below for using a numpy data table instead
```

Comment:
As mentioned above, I am against recommending use-cases for the API. Could we please try and keep it as simple as possible?

Reply:
Understood